src/hint_processor/builtin_hint_processor/hint_utils.rs

```diff
@@ -97,7 +97,7 @@ pub fn get_integer_from_var_name<'a>(
     ap_tracking: &ApTracking,
 ) -> Result<&'a BigInt, VirtualMachineError> {
     // let relocatable = get_relocatable_from_var_name(var_name, vm_proxy, ids_data, ap_tracking)?;
-    // vm_proxy.memory.get_integer(&relocatable)
+    // vm_proxy.get_integer(&relocatable)
```

Comment:
Can this comment be removed?

Reply:
yes, I didn't delete it because that comment was already in main.
[59249d9](https://github.com/lambdaclass/cairo-rs/pull/447/commits/59249d9c2dd77f9cafa4bffd331ce95898f4c10f).

Reply:
Thanks!